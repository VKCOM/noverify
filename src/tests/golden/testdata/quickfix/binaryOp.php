<?php

function binaryOp() {
    $x = $x + $y;
    $x = $x - $y;
    $x = $x * $y;
    $x = $x / $y;
    $x = $x % $y;
    $x = $x & $y;
    $x = $x | $y;
    $x = $x ^ $y;
    $x = $x << $y;
    $x = $x >> $y;
    $x = $x . $y;
    $x = $x ?? $y;
}
