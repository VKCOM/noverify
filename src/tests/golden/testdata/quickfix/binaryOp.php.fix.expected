<?php

function binaryOp() {
    $x += $y;
    $x -= $y;
    $x *= $y;
    $x /= $y;
    $x %= $y;
    $x &= $y;
    $x |= $y;
    $x ^= $y;
    $x <<= $y;
    $x >>= $y;
    $x .= $y;
    $x ??= $y;
}
